static/app/utils/discover/discoverQuery.tsx

```diff
@@ -11,7 +11,7 @@ import GenericDiscoverQuery, {
  * An individual row in a DiscoverQuery result
  */
 export type TableDataRow = {
-  [key: string]: React.ReactText;
+  [key: string]: string | number | null | (string | number | null)[];
```

Comment:
This change flows down to many components, so I will likely revert it and ts-expect error.